platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/IntReference.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2016-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.utility;
+
+import java.util.Objects;
+
+/** An object wrapper for an int that allows it to mutate, unlike {@link Integer}. */
+public class IntReference {
```

Comment:
- No thread safety is needed
- Looking at it now, it could have been avoided if it was decremented directly in the `ElectionRound`. I feel like its not worth changing in this PR. If you feel otherwise, I will make the change

Reply:
I guess I am conservative when it comes to creating one-off utility classes.   I'm fine with leaving it in with a disclaimer in the javadoc that it is known to not be thread safe and should only be used when there is a guarantee that only 1 thread will interact with it at a time.